examples/performance/in-vr/components/stats-in-vr.js

```diff
@@ -0,0 +1,38 @@
+/* global AFRAME */
+AFRAME.registerComponent('stats-in-vr', {
+  schema: {
+    // Number of frames for moving average.
+    frames: {default: 30}
+  },
+
+  init: function () {
+    this.renderTimes = [];
+  },
+
+  tick: function (t, dt) {
+    var color;
+    var fps;
+    var renderTimes = this.renderTimes;
+
+    renderTimes.push(dt);
+    if (renderTimes.length > this.data.frames) { renderTimes.shift(); }
+
+    fps = renderTimes.length / (renderTimes.reduce(sum) / 1000);
+
+    color = 'green';
+    if (fps < 80) {
```

Comment:
Oh, and I think if these ranges represented stable FPS, your ranges would be good, but they take into account frame drops.